packages/chat-component-bindings/src/errorBarSelector.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getLatestErrors } from './baseSelectors';
+import { ErrorType } from '@internal/react-components';
+import { createSelector } from 'reselect';
+
+/**
+ * Select active errors from the state for the `ErrorBar` component.
+ *
+ * Invariants:
+ *   - `ErrorType` is never repeated in the returned errors.
+ *   - Errors are returned in a fixed order by `ErrorType`.
+ */
+export const errorBarSelector = createSelector([getLatestErrors], (latestErrors): { activeErrors: ErrorType[] } => {
```

Comment:
So currently errors are all coming from stateful client, any future plan of catching errors from UI side?

Reply:
Not in the stateful client. We could save it "somewhere else", and the selector could merge the various sources.